src/extension/interop/ExtensionInterop.cpp

```diff
@@ -34,6 +39,19 @@ void DebugDrawerAddLine(DebugDrawer* drawerInstance, JVecF3* from, JVecF3* to, J
             *reinterpret_cast<godot::Color*>(colour));
 }
 
+bool ArrayMeshUnwrap(ArrayMeshVariant* mesh, float texelSize)
+{
+    godot::Variant* var = reinterpret_cast<godot::Variant*>(mesh);
+
+    godot::Object* obj = (godot::Object*)var;
+
+    obj->is_class("ArrayMesh");
```

Comment:
Shouldn't this fail if it is not the right class? And above there should be a check if the cast to object failed because someone could pass in a variant that just holds like the number 12.